### PR TITLE
Introduce the dual elliptic class and tests

### PR DESCRIPTION
**Pull request by alessiofumagalli**
- introduce the source for dual formulation problems
- remove the source term from the dual vem
- introduce the dual elliptic class
- introduce some test for the latter

### CODE REVIEW

File: src/porepy/numerics/elliptic.py [hunk 1]
```diff
@@ -43,7 +44,7 @@ class Elliptic():
             Returns Flux discretization object (E.g., Tpfa)
     grid(): Returns: the Grid or GridBucket
     data(): Returns: Data dictionary
-    split(name): Assignes the pressure self.p to the data dictionary at each
+    split(name): Assignes the solution self.solution to the data dictionary at each
```
Comment (rbe051) [inline src/porepy/numerics/elliptic.py]: should it be self.x and not self.solution ?

File: src/porepy/numerics/elliptic.py [hunk 2]
```diff
@@ -77,53 +83,115 @@ def reassemble(self):
...
+            self.split(self.pressure_name)
+        else:
+            self._data[self.pressure_name] = self.x
+
+    def discharge(self, discharge_name='discharge'):
+        if self.is_GridBucket:
+            fvutils.compute_discharges(self.grid(), self.physics,\
+                                       self.pressure_name)
+        else:
+            pass
```
Comment (rbe051) [inline src/porepy/numerics/elliptic.py]: I will fix this

File: src/porepy/numerics/elliptic.py [hunk 3]
```diff
@@ -77,53 +83,115 @@ def reassemble(self):
...
+                                      self.projected_discharge_name)
+        else:
+            discharge = self._data[self.discharge_name]
+            projected_discharge = self._flux_disc.project_u(self._gb,
+                                                          discharge, self._data)
+            self._data[self.projected_discharge_name] = projected_discharge
 
+#------------------------------------------------------------------------------#
 
 class EllipticData():
```
Comment (rbe051) [inline src/porepy/numerics/elliptic.py]: You don't need to change anything here?

File: src/porepy/numerics/elliptic.py (hunk 1 above)
Comment (alessiofumagalli) [reply src/porepy/numerics/elliptic.py]: yes, you're right.

File: src/porepy/numerics/elliptic.py (hunk 3 above)
Comment (alessiofumagalli) [reply src/porepy/numerics/elliptic.py]: well, nothing so far :)

**Pull request merged**